packages/credentials/lib/index.js

```diff
@@ -77,9 +77,11 @@ export class Credentials {
    */
   isDockCertsURL(url) {
```

Comment:
we might want to rename this to `isDockCertsCredentialUrl` or something

Reply:
doesn't have to be now, but...

Reply:
was thinking same